src/carma-1-tenth/check_distance_to_arrival.py

```diff
@@ -49,6 +49,9 @@ def check_distance_to_arrival(bag_dir):
                 # Store goal position
                 goal_positions[goal_position_count] = [strategy_params["destination"]["longitude"], strategy_params["destination"]["latitude"]]
                 goal_position_count += 1
+            elif topic == rviz_topic:
+                goal_position_count[goal_position_count] = [msg.pose.position.x, msg.pose.position.y]
```

Comment:
Is this supposed to be `goal_positions[goal_position_count]`?

Reply:
Yup, good catch!